server/configuration.go

```diff
@@ -24,6 +24,10 @@ import (
 // If you add non-reference types to your configuration struct, be sure to rewrite Clone as a deep
 // copy appropriate for your types.
 type configuration struct {
+	JitsiSettings jitsisettings
```

Comment:
Why this indirection here is needed? This will break the old configuration on upgrade.

Reply:
I tested the upgrade, it seems to work ok. The indirection is there because we wanted to keep the Jitsi settings for the custom admin console setting ui and add additional JaaS settings beside them. We wanted the Jitsi and JaaS features to be mutually exclusive. On upgrade the jitsi settings are copied into the new custom admin console settings. Is there another way to do this without using a custom admin console setting?